test/retry.js

```diff
@@ -71,6 +86,38 @@ test('falsy value prevents retries #2', async t => {
 	t.truthy(err);
 });
 
+test('custom retries', async t => {
+	let tried = false;
+	const err = await t.throws(got(`${s.url}/500`, {
+		throwHttpErrors: true,
+		retries: {
+			retry: iter => {
+				if (iter === 1) {
+					tried = true;
+					return 1;
+				}
+
+				return 0;
+			}, methods: [
+				'GET'
+			], statusCodes: [
+				500
+			]
+		}
+	}));
```

Comment:
Use the new `t.throws` API

Reply:
custom retries

  /home/szymon/Desktop/retries-patch/test/retry.js:93

   92:   let tried = false;
   93:   await t.throws(got(`${s.url}/500`, {
   94:     throwHttpErrors: true,

  The second argument to `t.throws()` contains unexpected properties

  Called with:

  {
    statusCode: 500,
  }

Reply:
Oh right... Never mind then.